src/App.tsx

```diff
@@ -20,17 +18,217 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    step: 3,
+    frameSize: 3,
+    itemWidth: 130,
+    animationDuration: 1000,
+    infinite: false,
+  };
+
+  setStep = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const curentValue = Number(event.currentTarget.value);
+
+    let newValue;
+
+    switch (true) {
+      case curentValue > 10:
+        newValue = 10;
+        break;
+      case curentValue < 0:
+        newValue = 0;
+        break;
+
+      default:
+        newValue = curentValue;
+        break;
+    }
+
+    this.setState({
+      step: newValue,
+    });
```

Comment:
it can be shorter)
fix in all set functions

```suggestion
    let curentValue = Number(event.currentTarget.value);

    switch (true) {
      case curentValue > 10:
        this.setState({ step: 10 });
        break;
      case curentValue < 0:
        this.setState({ step: 0 });
        break;

      default:
        this.setState({ step: curentValue });
    }
```